docs/source/getting_started.rst

```diff
@@ -0,0 +1,366 @@
+           file.write(text)
+       file.close()
+
+Our get_weather file should now look like this :
+
+.. code-block:: python
+   :caption: get_weather.py
+   :emphasize-lines: 4, 5, 28, 30, 31, 33
+
+
+   import requests
+   import remoulade
+   from remoulade.brokers.rabbitmq import RabbitmqBroker
+   from remoulade.results.backends import RedisBackend
+   from remoulade.results import Results
+
+
+   @remoulade.actor
+   def extract_city(city):
+      ...
+
+
+   @remoulade.actor()
+   def transform_city(response):
+       ...
+
+
+   @remoulade.actor
+   def load_city(text):
+      ...
+
+
+   @remoulade.actor
+   def load_cities(city_lines):
+       ...
+
+
+   result_backend = RedisBackend()
+   rabbitmq_broker = RabbitmqBroker()
+   result_time_limit_ms = 10 * 60 * 1000 # 10 mn
+   rabbitmq_broker.add_middleware(Results(backend=result_backend, store_results=True, result_ttl=result_time_limit_ms))
+   remoulade.set_broker(rabbitmq_broker)
+   remoulade.declare_actors([extract_city, transform_city, load_city, load_cities])
+
+You will need to restart your Remoulade workers once again after adding this new function.
+Two new things are happening here. First we instantiate a |RedisResBackend| Result Backend that will allow us to store our data in Redis. Then we add a |Results| middleware that will store our results in this backend, with a defined time limit.
+
+Our objective can now be achieved by running the following commands::
+
+   >>> from get_weather import extract_city, transform_city, load_cities
+   >>> from remoulade import pipeline, group
+   >>> cities = ['Paris', 'Lyon', 'Marseille', 'Lille', 'Bastia']
+   >>> grp = group([extract_city.message(city) | transform_city.message() for city in cities])
+   >>> weather_etl_pipeline = grp | load_cities.message()
+   >>> weather_etl_pipeline.run()
+
+Scheduling Messages
+-------------------
+
+Remoulade includes a scheduler that allows to periodically run actors. We can use it here to keep our weather data updated.
+
+Let's get back to our ``get_weather`` function and add a scheduler to update the weather data every 10 seconds.
+
+.. code-block:: python
+   :caption: get_weather.py
+   :emphasize-lines: 4, 21, 22, 24
+
+   import requests
+   import remoulade
+   from remoulade.brokers.rabbitmq import RabbitmqBroker
+   from remoulade.scheduler import ScheduledJob, Scheduler
+
+
+   @remoulade.actor
+   def get_weather(city):
+        url = f"https://goweather.herokuapp.com/weather/{city}"
+
+        response = requests.get(url).json()
+
+        file = open('weather_db.txt', 'a')
+        text = f"{city}: {response['temperature']}\n"
+        file.write(text)
+        file.close()
+
+
+   rabbitmq_broker = RabbitmqBroker()
+   remoulade.set_broker(rabbitmq_broker)
+   scheduler = Scheduler(rabbitmq_broker, [ScheduledJob(actor_name="get_weather", args=("Paris",), interval=10)])
+   remoulade.set_scheduler(scheduler)
+   remoulade.declare_actors([get_weather])
+   scheduler.start()
```

Comment:
I think what you can do is to remove the start line and then do 
```python
scheduler = remoulade.get_scheduler()
scheduler.start()
```
in a python shell. What do you think?

Reply:
This doesn't seem to be possible :
remoulade.errors.NoScheduler: Scheduler not found, are you sure you called set_scheduler(scheduler) ?